frontend/src/lib/components/Cards/TextCard/TextCardModal.tsx

```diff
@@ -67,9 +71,7 @@ export function TextCardModal({
             >
                 <PayGateMini feature={AvailableFeature.DASHBOARD_COLLABORATION}>
                     <Field name="body" label="">
-                        {({ value, onChange }) => (
-                            <LemonTextMarkdown data-attr={'text-card-edit-area'} value={value} onChange={onChange} />
-                        )}
+                        <LemonTextMarkdown data-attr={'text-card-edit-area'} />
```

Comment:
oh cool, value and onChange automatically propagate? 🎸

Reply:
Yup, `Field` just clones the child component with `onChange` and `value` slapped on before rendering. Had to make those props optional on `LemonTextMarkdown` for TypeScript not to complain, but what do.